examples/weight_compression/openvino/tiny_llama/main.py

```diff
@@ -0,0 +1,211 @@
+import datetime
+import itertools
+from functools import partial
+from pathlib import Path
+from typing import Callable, Iterable, Optional, TypeVar
+
+import numpy as np
+import openvino as ov
+from datasets import load_dataset
+from optimum.intel import OVModelForCausalLM
+from transformers import AutoTokenizer
+from whowhatbench import Evaluator
+
+import nncf
+from nncf.common.logging import nncf_logger
+
+DataItem = TypeVar("DataItem")
+ModelInput = TypeVar("ModelInput")
+
+ROOT = Path(__file__).parent.resolve()
+MODEL_PATH = ROOT / "compressed_model.xml"
+
+COMPRESSION_MODE = nncf.parameters.CompressWeightsMode.INT4_SYM
+MAX_DROP = 0.2
```

Comment:
Let's place the comment that we consider the following range of parameters: `group_size` - [64, 128], `ratio` - [0.5,...,1.0]